test/studies/prk/stencil/COMPOPTS

```diff
@@ -0,0 +1,2 @@
+--fast -sassertNoSlicing
```

Comment:
COMPOPTS is what's used for correctness testing. You probably don't want to be throwing performance oriented flags here